manifests/collect_events.pp

```diff
@@ -0,0 +1,6 @@
+cron { 'collectpeapi':
+  ensure  => 'present',
+  command => '/etc/puppetlabs/code/environments/production/modules/splunk_hec/scripts/collect_api_events.rb',
```

Comment:
Is there a settings variable we can use for this? This stuff isn't always in `/etc`. Can we raise a ticket to change this later. I know we want to just get this running for now.